theHarvester/__main__.py

```diff
@@ -125,7 +125,7 @@ def store(search_engine: Any, source: str, process_param: Any = None, store_host
                 for usr in sorted(list(set(people_list))):
                     print(usr)
         if store_links:
-            links = linkedin_links_search.get_links()
+            links = search_engine.get_links()
             db.store_all(word, links, 'name', 'linkedin')
```

Comment:
Can you get it to not be linkedin as previously mentioned other modules that get links would reference linkedin if this is used so can you change it to the engine name that is requesting to save the links